Pipfile

```diff
@@ -12,6 +12,8 @@ whitenoise = "6.2.0"
 gunicorn = "20.1.0"
 djangorestframework = "3.13.1"
 drf-yasg = "1.20.0"
+djangorestframework-simplejwt = "5.2.0"
+types-six = "*"
```

Comment:
remove types-six use smart bytes/ smart str django in built functions

Reply:
Done